cfgov/jinja2/v1/_includes/organisms/mortgage-chart.html

```diff
@@ -99,7 +99,7 @@
                         </select>
                     </div>
                     <p class="m-field-helper-text u-mt15" id="mp-line-chart-state-metro-helper-text" style="display:none">
-                      Some metro areas cover multiple areas.</a>
+                      Some metro areas cover multiple states.</a>
```

Comment:
Just occurs to me that "cover" may not be the right word, since cover often is construed as completely covering.
"States" is clearly an improvement over "areas," so this shouldn't stop the PR, but I'd propose these alternatives:
- "Some metro areas cross state boundaries."
- "Some metro areas touch multiple states."

Reply:
Cool. Can you bring that to our product owner's attention tomorrow? She wrote and is responsible for the content.